resources/make_test_resources.sh

```diff
@@ -12,24 +12,32 @@ SOURCE=$(readlink -f "$0")
 TEST_RESOURCE_DIR="$(dirname "$SOURCE")"
 cd $TEST_RESOURCE_DIR
 
-# If user doesn't provide a value for a variable, use the default one (which is
-# the one used by the tests).
-KERNEL_DIR=${KERNEL_DIR:="/tmp/vmlinux_busybox"}
-DEB_DIR=${DEB_DIR:="/tmp/ubuntu-focal"}
-DISK_DIR=${DISK_DIR:="/tmp/ubuntu-focal-disk"}
+# If user doesn't provide a value for a variable, use the default one.
+# We build the images in /tmp, so they don't pollute the vmm-reference
+# repository, and then move them at the locations expected by the tests.
+TMP_KERNEL_DIR=${TMP_KERNEL_DIR:="/tmp/vmlinux_busybox"}
+TMP_DEB_DIR=${TMP_DEB_DIR:="/tmp/ubuntu-focal"}
+DEST_KERNEL_DIR=${DEST_KERNEL_DIR:="$TEST_RESOURCE_DIR/kernel"}
+DEST_DISK_DIR=${DEST_DISK_DIR:="$TEST_RESOURCE_DIR/disk"}
 
 arch=$(uname -i)
 if [[ $arch = "x86_64" ]]; then
-    ./kernel/make_kernel_busybox_image.sh -f elf -k vmlinux-hello-busybox -w $KERNEL_DIR -j 2
-    ./kernel/make_kernel_busybox_image.sh -f elf -k vmlinux-hello-busybox -w $KERNEL_DIR -j 2 -h
-    ./kernel/make_kernel_busybox_image.sh -f bzimage -k bzimage-hello-busybox -w $KERNEL_DIR -j 2
-    ./kernel/make_kernel_busybox_image.sh -f bzimage -k bzimage-hello-busybox -w $KERNEL_DIR -j 2 -h
-    ./kernel/make_kernel_image_deb.sh -f elf -k vmlinux-focal -w $DEB_DIR -j 2
-    ./kernel/make_kernel_image_deb.sh -f elf -k vmlinux-focal -w $DEB_DIR -j 2 -h
-    ./kernel/make_kernel_image_deb.sh -f bzimage -k bzimage-focal -w $DEB_DIR -j 2
-    ./kernel/make_kernel_image_deb.sh -f bzimage -k bzimage-focal -w $DEB_DIR -j 2 -h
-    ./disk/make_rootfs.sh -d /tmp/ubuntu-focal/linux-5.4.81/deb/ -w $DISK_DIR -o rootfs.ext4
+    ./kernel/make_kernel_busybox_image.sh -f elf -k vmlinux-hello-busybox -w $TMP_KERNEL_DIR -j 2
+    ./kernel/make_kernel_busybox_image.sh -f elf -k vmlinux-hello-busybox -w $TMP_KERNEL_DIR -j 2 -h
+    ./kernel/make_kernel_busybox_image.sh -f bzimage -k bzimage-hello-busybox -w $TMP_KERNEL_DIR -j 2
+    ./kernel/make_kernel_busybox_image.sh -f bzimage -k bzimage-hello-busybox -w $TMP_KERNEL_DIR -j 2 -h
+    mv $TMP_KERNEL_DIR/linux-5.4.81/vmlinux-hello-busybox $TMP_KERNEL_DIR/linux-5.4.81/vmlinux-hello-busybox-halt \
```

Comment:
Can you separate these 2 by a whitespace so it's more clear that you are building them, and then move them to where they are expected to be found by the vmm-reference tests? It would also be nice to add a comment that we're moving the resources to make it easier to upload the whole `resource` folder to S3.

Reply:
Yep, sorry, added more comments.